apps/root/src/pages/dca/create-position/components/swap/index.tsx

```diff
@@ -364,7 +364,19 @@ const Swap = ({ currentNetwork, yieldOptions, isLoadingYieldOptions, handleChang
         shouldEnableYield ? toYield?.tokenAddress : undefined,
         signature
       );
-      trackEvent('DCA - Create position submitted');
+      try {
+        trackEvent('DCA - Create position submitted', {
```

Comment:
You have added a try catch in the upper level, I don't think its necessary here

Reply:
Yeah but it parses the usd price at some point, and honestly, I was scared that it could explode through that, just being precacious